code/modules/gear_presets/uscm_visiting_officers.dm

```diff
@@ -0,0 +1,261 @@
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/marine/officer/bridge(new_human), WEAR_BODY)
+	new_human.equip_to_slot_or_del(new back_item(new_human), WEAR_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/pistol/command(new_human), WEAR_L_STORE)
+	new_human.equip_to_slot_or_del(new sidearmpath(new_human), WEAR_WAIST)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large(new_human), WEAR_R_STORE)
+
+	if(kit)
+		new_human.equip_to_slot_or_del(new kit(new_human), WEAR_IN_BACK)
+
+/datum/equipment_preset/uscm_visitor/co/ltcol
+	name = "USCM Observer (Lt. Col.)"
+	paygrades = list(PAY_SHORT_MO5 = JOB_PLAYTIME_TIER_0)
+
+/datum/equipment_preset/uscm_visitor/co/col
+	name = "USCM Observer (Colonel)"
+	paygrades = list(PAY_SHORT_MO6 = JOB_PLAYTIME_TIER_0)
+
+/datum/equipment_preset/uscm_visitor/xo
+	name = "USCM Junior Observer (Captain)"
+	paygrades = list(PAY_SHORT_MO3 = JOB_PLAYTIME_TIER_0)
+	assignment = JOB_USCM_OBSV_JNR
+	job_title = JOB_USCM_OBSV_JNR
+	role_comm_title = "USCM J-Obs"
+	minimap_icon = "xo"
+
+	minimum_age = 27
+	skills = /datum/skills/XO
+
+/datum/equipment_preset/uscm_visitor/xo/New()
+	. = ..()
+	access = get_access(ACCESS_LIST_MARINE_MAIN)
+	access |= ACCESS_MARINE_DATABASE_ADMIN
+
+/datum/equipment_preset/uscm_visitor/xo/load_gear(mob/living/carbon/human/new_human)
+	var/back_item = /obj/item/storage/backpack/satchel
+	if (new_human.client && new_human.client.prefs && (new_human.client.prefs.backbag == 1))
+		back_item = /obj/item/storage/backpack/marine
+
+	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/mcom/cdrcom/xo(new_human), WEAR_L_EAR)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/marine/officer/bridge(new_human), WEAR_BODY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/jacket/marine/service(new_human), WEAR_JACKET)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/dress(new_human), WEAR_FEET)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/marine/peaked/service(new_human), WEAR_HEAD)
+	new_human.equip_to_slot_or_del(new back_item(new_human), WEAR_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large(new_human), WEAR_L_STORE)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large(new_human), WEAR_R_STORE)
+	new_human.equip_to_slot_or_del(new /obj/item/device/binoculars/range(new_human), WEAR_L_STORE)
+
+/datum/equipment_preset/uscm_visitor/honor_guard_lead
+	name = "USCM Lead Honor Guard (Observer)"
+	assignment = JOB_USCM_HG_SL
+	job_title = JOB_USCM_HG_SL
+	paygrades = list(PAY_SHORT_ME7 = JOB_PLAYTIME_TIER_0)
+	role_comm_title = "USCM HG-L"
+	skills = /datum/skills/honor_guard/vet
+	idtype = /obj/item/card/id/silver
+
+	minimap_icon = "vhg"
+	minimap_background = "background_mp"
+
+/datum/equipment_preset/uscm_visitor/honor_guard_lead/load_gear(mob/living/carbon/human/new_human)
```

Comment:
this preset doesn't have any med supplies to do first aid on the person they're protecting; nor any MP gear to deal with non-emergency rabble harassing them. maybe the pouch slots?